jenkins-jobs/mongodb-matrix-test.yaml

```diff
@@ -0,0 +1,53 @@
+# Job definition to test MongoDB connector against MongoDB
+---
+- job:
+    project-type: matrix
+    name: debezium-mongodb-matrix-test
+    display-name: Debezium MongoDB Test Matrix
+    description: Executes tests for MongoDB Connector with MongoDB matrix
+    execution-strategy:
+      sequential: true
+    axes:
+      - axis:
+          type: user-defined
+          name: MONGODB_VERSION
+          values:
+            - 3.2
+            - 3.4
```

Comment:
The reason you didn't put 3.6 is because it's the default in pom?
If so I'd say we could change the pom from `3.6.3` to `3.6` only, because now we have no way to tell if new releases of 3.6 are broken

Reply:
@renatomefi IMHO there is already 3.6 only in pom